src/pages/RegisterAndLogin/Components/PassRqrmt.tsx

```diff
@@ -0,0 +1,96 @@
+import { Flex, Icon, Text } from "@chakra-ui/react";
+import RedTickIcon from "../../../icons/RedTickIcon";
+
+const PassRqrmt = ({
+  formData
+}: {
+  formData: {
+    email: string;
+    password: string;
+  };
+}) => {
+  const { password } = formData;
+  const hasPsValue = password;
+  const hasPsassword7chars = password.length > 6;
+  const hasPsNumber = /\d/.test(password);
+  const hasPsUpperCase = /[A-Z]/.test(password);
+  const hasPsSmallCase = /[a-z]/.test(password);
+  const hasPsSymbol = /\W|_/g.test(password);
+  return (
+    <Flex
+      flexDirection="column"
+      w="350px"
+      h="185px"
+      boxShadow="xs"
+      p="1px"
+      rounded="md"
+      bg="white"
+    >
+      <Flex flexDirection="row" alignItems="flex-start">
```

Comment:
The default direction value is already row. No need to add it. And also there is no need to align items. I removed it nothing changed. Think using Hstack

Reply:
I tried Hstack and the first line broke the align then I backed to Flex and removed unnecessary flexDirectio and alignItems.
![Screenshot 2023-03-13 230312](https://user-images.githubusercontent.com/61114943/224819597-1bac6a70-fec3-46cb-98a6-84fd6e8ce264.png)